edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/wizards/newproject/WPIRobotJavaProjectCreator.java

```diff
@@ -38,10 +38,10 @@ public String getPackageName() {
 	@Override
 	public Map<String, String> getValues() {
 		Map<String, String> vals = new HashMap<String, String>();
-		vals.put("$project", projectName);
-		vals.put("$package", packageName);
+		vals.put("(\\$project)", projectName);
```

Comment:
You shouldn't have to wrap the regex in a group for it to match. Is it needed here?

Reply:
Yeah, it wasn't needed, so fixed